asyncscope.md

```diff
@@ -1270,6 +1391,16 @@ should have a name that conveys danger.
 
 alternatives: `complete()`, `close()`
 
+## "Token" as in `async_scope_token` and `counting_scope::token`
+
+The first several revisions of this paper did not separate the responsibilities of an async scope into the scope and its
```

Comment:
Maybe add a "Split counting_scope into scope and token" blurb into the R3 section of the changes?